subsys/testsuite/ztest/src/ztress.c

```diff
@@ -319,6 +320,14 @@ int ztress_execute(struct ztress_context_data *timer_data,
 		return -EINVAL;
 	}
 
+	/* Skip test if system clock is set too high compared to CPU frequency.
+	 * It can happen when system clock is set globally for the test which is
+	 * run on various platforms. 1000 factor is an arbitrary value.
+	 */
+	if ((sys_clock_hw_cycles_per_sec() / CONFIG_SYS_CLOCK_TICKS_PER_SEC) < 1000) {
+		ztest_test_skip();
+	}
+
```

Comment:
IIRC, there are a bunch of Nordic boards where hardware clock and sys ticks are the same (32.768kHz?). This would result in stress test not running on those boards. Someone might want to test on those boards to make sure logging works for them.

Reply:
Right, i thought that sys_clock_hw_cycles_per_sec returns CPU frequency (64MHz in case of nrf52). This is not the case. Need to find other way then.

Reply:
I came up with a runtime test that checks how many system tick it takes to run a dummy loop. If too many ticks elapsed, it means that sys clock is too high (compared to cpu clock). Test is skipped in that case.